demos/sslecho/README.md

```diff
@@ -24,3 +24,49 @@ The cert.pem and key.pem files included are self signed certificates with the
 "Common Name" of 'localhost'.
 
 Best to create the 'pem' files using an actual hostname.
+
+Encrypted Client Hello (ECH) Variant
+====================================
+
+``echecho.c`` implements the same functionality but demonstrates minimal code
+changes needed to use ECH. The ``echecho`` binary has the same user interface
+discussed above but enables ECH for the connection, based on hard-coded ECH
+configuration data. A real server would load file(s), and a real client would
+acquire an ECHConfigList from the DNS.
+
+All that's required to use ECH is for the server to enable ECH via
+``SSL_CTX_ech_server_enable_buffer()`` and for the client to do
+similarly via ``SSL_CTX_ech_set1_echconfig()``. Both client and
+server check and print out the status of ECH using ``SSL_ech_get_status()``,
+but that's optional.
```

Comment:
This does not look like it is up to date with the API changes?

Reply:
good catch, updated